File: Java/백준/Silver/11371. The Big Eye in the Sky/The Big Eye in the Sky.java
import java.util.Scanner;
import java.lang.Math;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        while(true){
            double x = in.nextDouble();
            double y = in.nextDouble();
            if (x==0 && y==0){
                break;
            }
            if (x==0){
                System.out.println(90);
            }
            else{
                System.out.println(Math.round(180*Math.atan(y/x)/Math.PI));
            }
        }
    }
}

File: C#/백준/Silver/11371. The Big Eye in the Sky/The Big Eye in the Sky.cs
using System;

namespace q{
    class w{
        static void Main(){
            while(true){
                float[] aa = Array.ConvertAll(Console.ReadLine().Split(),float.Parse);
                if (aa[0]==aa[1] && aa[0]==0){
                    break;
                }
                if (aa[0]==0){
                    Console.WriteLine(90);
                }
                else{
                    Console.WriteLine(Math.Round(180*Math.Atan(aa[1]/aa[0])/Math.PI));
                }
            }
        }
    }
}